package.json

```diff
@@ -1,95 +1,121 @@
 {
-	"name": "dotnet",
```

Comment:
@strakamichal Can you go over the PR again and make sure that your line ending settings are correct in git and that your patches do not change every line ending even if they do not touch the file?

Reply:
Also please check that you do not do reformatting :)